resources/schemas/documentContent.json

```diff
@@ -0,0 +1,38 @@
+{
+  "additionalProperties": true,
+  "properties": {
+    "id": {
+      "type": "string"
+    },
+    "organizationId": {
+      "type": "string"
+    },
+    "createdAt": {
+      "type": "string"
+    },
+    "updatedAt": {
+      "type": "string"
+    },
+    "archivedAt": {
+      "type": ["null", "string"]
+    },
+    "content": {
+      "type": ["null", "string"]
+    },
+    "contentData": {
+      "type": ["null", "object"]
+    },
```

Comment:
This field is deprecated and will soon be removed. I don't think we should introduce it to Airbyte connector?